### ISSUE TITLE
expect() with no actual expectations

### ISSUE THREAD

**Issue by alecxe** [tlvince/eslint-plugin-jasmine#23]
The problem and a custom eslint rule implementation can be found here: http://stackoverflow.com/questions/33470377/expect-with-no-actual-expectations.

To summarize, do you think we can add a new jasmine eslint rule that would catch:

* more than 1 argument passed to `expect()`
* there was nothing called on `expect()`

Example violations:

```javascript
expect(page.filters.fromDateLabel.getText(), "After");
expect("After");
expect();
```

Please let me know if it makes sense to make this rule a part of `eslint-plugin-jasmine` and I'll prepare a PR. Thanks!

**Comment by tlvince** [comment]
That sounds like a useful feature. Happy to include.

**Comment by tlvince** [comment]
Closed by #24.
